relay/src/lib.rs

```diff
@@ -15,22 +15,14 @@
 use std::time::Duration;
 
 use anyhow::{anyhow, bail, Context, Result};
-use bonsai_sdk::alpha::{responses::SnarkProof, Client, SdkErr};
+use bonsai_sdk::alpha::{responses::SnarkReceipt, Client};
 use risc0_build::GuestListEntry;
-use risc0_zkvm::{
-    Executor, ExecutorEnv, MemoryImage, Program, Receipt, ReceiptMetadata, MEM_SIZE, PAGE_SIZE,
-};
+use risc0_zkvm::{Executor, ExecutorEnv, MemoryImage, Program, Receipt, MEM_SIZE, PAGE_SIZE};
 
 /// Result of executing a guest image, possibly containing a proof.
 pub enum Output {
-    Execution {
-        journal: Vec<u8>,
-    },
-    Bonsai {
-        journal: Vec<u8>,
-        receipt_metadata: ReceiptMetadata,
-        snark_proof: SnarkProof,
-    },
+    Execution { journal: Vec<u8> },
+    Bonsai { snark_proof: SnarkReceipt },
```

Comment:
```suggestion
    Bonsai { snark_receipt: SnarkReceipt },
```

Reply:
Thanks! Fixed in: 7ba6cb4ab87ab9ef8f758a5eee457508859e191f

Reply:
I mean: ce7ce12348455a8d3a0512d60432c06eebb39a11